src/components/AppNavigationForm.vue

```diff
@@ -126,6 +148,23 @@ export default {
 			return t('forms', 'New form')
 		},
 
+		/**
+		 * Return expiration details for subtitle
+		 */
+		formSubtitle() {
+			if (this.form.expires) {
+				const expireDate = moment(this.form.expires, 'X').fromNow()
+				if (this.isExpired) {
+					return t('forms', 'Expired {relativeDate}', { relativeDate: expireDate })
+				}
+				return t('forms', 'Expires {relativeDate}', { relativeDate: expireDate })
```

Comment:
Also a small code-beauty: With according naming, you can directly pass the variable without separately assigning.
```suggestion
				const relativeDate = moment(this.form.expires, 'X').fromNow()
				if (this.isExpired) {
					return t('forms', 'Expired {relativeDate}', { relativeDate })
				}
				return t('forms', 'Expires {relativeDate}', { relativeDate })
```